src/entrypoints/main/router/pages/contact-page/index.tsx

```diff
@@ -1,29 +1,53 @@
-import React, { FC, memo, useEffect } from 'react';
+import React, { FC, memo } from 'react';
 import { compose } from 'redux';
 
-import withPage, {
-  Props as CmsArticleProps
-} from '../../../../../components/with-cms-page';
-
-import Article from '../../../../../components/article';
 import Root from '../../../../../components/root';
+import Spacing from '../../../../../components/spacing';
+import env from '../../../../../env';
+import { useGetContactsQuery } from '../../../../../services/api/generated/cms/graphql';
+import SC from './styled';
 
-const articleId = '382bcca1-3b63-4a03-9719-bfb1810464e0';
-
-interface Props extends CmsArticleProps {}
+const { STRAPI_API_HOST } = env;
 
-const FindDataPage: FC<Props> = ({
-  cmsPage,
-  cmsPageActions: { getCmsPageRequested: getCmsPage, resetCmsPage }
-}) => {
-  useEffect(() => {
-    getCmsPage(articleId);
-    return () => {
-      resetCmsPage();
-    };
-  }, []);
+const FindDataPage: FC = () => {
+  const { data: { contactPage } = {} } = useGetContactsQuery();
 
-  return <Root invertColor>{cmsPage && <Article article={cmsPage} />}</Root>;
+  return (
+    <Root>
+      <SC.Header>
+        <SC.Container>
+          <SC.Title>{contactPage?.title}</SC.Title>
+          {contactPage?.content?.[0]?.__typename ===
+            'ComponentBasicParagraph' && (
+            <SC.Ingress>{contactPage.content[0].content}</SC.Ingress>
+          )}
+        </SC.Container>
+      </SC.Header>
+      <SC.Container>
+        <SC.ContactCardContainer>
+          {contactPage?.content?.map(
+            contact =>
+              contact?.__typename === 'ComponentBasicContact' && (
+                <SC.ContactCard>
+                  {contact?.image?.[0]?.url && (
+                    <SC.ContactImage
+                      src={`${STRAPI_API_HOST}${contact.image[0].url}`}
+                    />
+                  )}
+                  <SC.ContactName>{contact.name}</SC.ContactName>
+                  <p>{contact.jobTitle}</p>
+                  <Spacing.S8 />
```

Comment:
Hvorfor bruke spacing comp isteden for ha padding i styled component?

Reply:
Vi gjør ikke dette noe annet sted egentlig, så jeg hadde satt padding i SC.

Reply:
Et alternativ til Spacer er "gap" for Flexbox, det skal vel ha støtte i de fleste browsere nå?